### ISSUE TITLE
Source Push errors aren't visible.

### ISSUE THREAD

**Issue by FishOfPrey** [wadewegner/deploy-to-sfdx#12]
I'm attempting to deploy https://github.com/muenzpraeger/salesforce-einstein-platform-apex to a scratch org with the direct deploy button.

It gives me the following output:

> Finished. You have deployed the app to your scratch org!

> 7 ) Removed temp files and cleaned up.
> 6 ) Generated a login url: https://momentum-data-1744-dev-ed.cs100.my.salesforce.com/secur/frontdoor.jsp?sid=00D0q0000000001!ARwAQC32j9otrbvI._NotMyRealSessionId_mqN0XVw2JA66bNuGsBH2chbh
> 5 ) Apex tests: .
> 4 ) Pushed  source files.
> 3 ) Successfully created scratch org: 00D0q0000000001EAU, username: test-s97bv57q1234@muenzpraeger_-_rené_winkelmeyer.net.
> 2 ) Successfully cloned https://github.com/muenzpraeger/salesforce-einstein-platform-apex.
> 1 ) Didn't find a .salesforcedx.yaml file. Using defaults:
	assign-permset: false
	permset-name: 
	delete-scratch-org: false
	run-apex-tests: false
	scratch-org-def: config/project-scratch-def.json
	show-scratch-org-url: true

Which gave ever indication that the deployment was successful. Yet when I log into my new ~~scratch~~ burner org I don't see any of the expected metadata from the source repo.

I've discussed it with @muenzpraeger, and the theory is that his source requires a Winter '17 org. My scratch orgs are currently being created on Summer '17.

I tried deploying to my burner org from the CLI directly and got the following errors:

> PROJECT PATH  ERROR
────────────  ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
N/A           No COMPONENT named markup://lightning:accordionSection found : [markup://c:EinsteinPlaygroundIntroduction]
N/A           No COMPONENT named markup://lightning:accordionSection found : [markup://c:EinsteinPlayground, markup://c:EinsteinPlaygroundIntroduction]

René commented that these will be the Winter '17 specific components.

So a couple of observations:

1. These errors aren't visible when using the deploy button.
2. There is no mechanism in force:source:push to block the deployment if the target org doesn't have the required API version. Raised this on the [Success community](https://success.salesforce.com/0D53A00003FLMiE).

**Comment by wadewegner** [comment]
Yes, definitely looks like an error. When pushing metadata, it should tell you the # of metadata files, like:
```
5) Pushed 6 source files.
```
I'll take a look to see why it's not generating an error on push.

**Comment by wadewegner** [comment]
FYI, for now I removed Rene's repo from the chooser.

**Comment by muenzpraeger** [comment]
LOL

**Comment by andyforandy** [comment]
I am facing similar issue, any luck with a fix?
